LibTest/js_interop/JSExport/jsExport_A06_t07.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Only concrete instance members can and will be wrapped, and it's
+/// an error to annotate other members with this annotation.
+///
+/// @description Checks that it is a compile-time error if an old-fashioned
+/// function type is annotated with `JSExport()`.
+/// @author sgrekhov22@gmail.com
+/// @issue 61093
+
+import 'dart:js_interop';
+
+@JSExport()
+typedef void Foo();
+//           ^^^
+// [analyzer] unspecified
+// [web] unspecified
```

Comment:
This expectation is good: It fails for now, and we may not be able to fix it, but right now we just want to expect all the diagnostic messages that we _want_ to have.

Reply:
It should be possible for backends to report errors on typedef *declarations* but not uses through a typedef. So it should be possible to report an error here.